tensorflow/tools/dist_test/python/mnist_replica.py

```diff
@@ -183,13 +148,19 @@ def main(unused_argv):
       cross_entropy = -tf.reduce_sum(y_ *
                                      tf.log(tf.clip_by_value(y, 1e-10, 1.0)))
 
+      if FLAGS.sync_replicas:
+        if FLAGS.replicas_to_aggregate is None:
+          replicas_to_aggregate = FLAGS.num_workers
+        else:
+          replicas_to_aggregate = FLAGS.replicas_to_aggregate
+
```

Comment:
This `if FLAGS.sync_replicas:` block can be combined with the one below (on L158&ndash;164).